nntrainer/src/optimizer.cpp

```diff
@@ -85,7 +85,7 @@ void Optimizer::apply_gradients(
 
   float ll = popt.learning_rate;
   if (popt.decay_steps != -1) {
-    ll = ll * pow(popt.decay_rate, (iteration / popt.decay_steps));
+    // ll = ll * pow(popt.decay_rate, (iteration / popt.decay_steps));
   }
```

Comment:
Do we have to comment out this learning rate decay?

Reply:
This is waiting for #253 as the earlier default value of `decay_steps` of `0` made a loss as `nan`

Reply:
guess this can be reverted now?

Reply:
Removed unnecessary changes.